ci-operator/jobs/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-release-4.11-presubmits.yaml

```diff
@@ -113,7 +113,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )helm-lint,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Also unrelated (must be SRO misconfiguration)?

Reply:
Same as above, customizations are just out of sync. I'll clean these up.